src/components/BrowseSkill/BrowseSkill.js

```diff
@@ -319,6 +319,10 @@ export default class BrowseSkill extends React.Component {
           data.sort();
           languages = [];
           for (let i = 0; i < data.length; i++) {
+            if (data[i] === '0' || data[i] === 'xx') {
```

Comment:
Please check if they are the same type, otherwise === would always return false.

Reply:
Done!

Reply:
@mkfeuhrer, @tabesin meant to check the data-type of `data[i]`, if it is the same as a string, else with using `===` as comparator, the statement would always evaluate to false, as it also compares data-type.

Reply:
@rossoskull  @tabesin  I checked and ensured the datatypes of both by debugging . Since both are string === works fine here.